css-checks/src/main/resources/org/sonar/l10n/css/rules/css/formatting.html

```diff
@@ -0,0 +1,79 @@
+<p>Shared coding conventions make it possible for a team to collaborate efficiently. This rule allows to check compliance with formatting standards.</p>
+<h1>Block</h1>
+<ul>
+  <li>Opening curly brace should be placed on the same line as the block declaration</li>
+  <li>Closing curly brace should be placed on its own line</li>
+</ul>
+<h2>Noncompliant Code Example</h2>
+<pre>
+.mybox
+{                        /* Noncompliant: the opening curly brace should be moved to the previous line */
+  color: red;
+}
+
+.mybox { color : red; } /* Noncompliant: the code following the opening curly brace should be moved to the next line */
+
+.mybox {
+  color: red;
+} .mybox {              /* Noncompliant: the code following the closing curly brace should be moved to the next line */
+  color: blue;
+}
+
+.mybox { color : red; }  /* Noncompliant: the closing curly brace should be moved to the next line */
```

Comment:
Indeed, it would be more meaningful. Thanks!
Fixed in https://github.com/racodond/sonar-css/commit/84914bd432a3a272c94a388a0110bce9c5ab04be